### PR TITLE
Make changes for CS2101

### PR DESCRIPTION
**Pull request by vishandi**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -140,13 +140,8 @@ Format: `add n/NAME p/PHONE_NUMBER [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL] [a/ADDRES
...
-* `ADDRESS`, if specified, should only contain alphanumeric characters, spaces, and the following punctuations: **!"#$&'()\*+,-.:;<=>?@**. It should not be blank and be at most 80 characters long.
-* `EMAIL`, if specified, should be a **valid** email with length not exceeding 60 characters.
-* **Valid** `EMAIL` are emails that are of the format *local-part@domain* and adhere to the following constraints:
-  * The local-part should only contain alphanumeric characters and the following special characters: **+_.-**. It cannot start or end with any special characters and cannot have consecutive special characters.
-  * The domain part is made up of domain labels separated by periods (e.g. domain-label1.domain-label2.com). It must end with a domain label at least 2 characters long, each domain label starts and end with alphanumeric characters, and each domain label consists of only alphanumeric characters, separated only by hyphens, if any.
+* `EMAIL`, if specified, should be a **valid** email.
+* Similarly, `TELEGRAM` and `GITHUB`, if specified, should be a **valid** usernames as well.
```
Comment (markbiju) [inline docs/UserGuide.md:144]:
```suggestion
* Similarly, `TELEGRAM` and `GITHUB`, if specified, should be **valid** usernames as well.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -181,18 +176,14 @@ Format: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL
...
-* `EMAIL`, if specified, should be a **valid** email with length not exceeding 60 characters.
-* **Valid** `EMAIL` are emails that are of the format *local-part@domain* and adhere to the following constraints:
-    * The local-part should only contain alphanumeric characters and the following special characters: **+_.-**. It cannot start or end with any special characters and cannot have consecutive special characters.
-    * The domain part is made up of domain labels separated by periods (e.g. domain-label1.domain-label2.com). It must end with a domain label at least 2 characters long, each domain label starts and end with alphanumeric characters, and each domain label consists of only alphanumeric characters, separated only by hyphens, if any.
+* `NAME` should only contain alphanumeric characters and spaces. It should not be blank and be at most 30 characters long.
+* `PHONE_NUMBER` should only contain numeric characters, and should be 3-15 digits long.
+* `EMAIL`, if specified, should be a **valid** email.
+* Similarly, `TELEGRAM` and `GITHUB`, if specified, should be a **valid** usernames as well.
```
Comment (markbiju) [inline docs/UserGuide.md:182]:
```suggestion
* Similarly, `TELEGRAM` and `GITHUB`, if specified, should be **valid** usernames as well.
```

**Review by markbiju — COMMENTED** [review]
Overall just minor changes from me, other than that LGTM!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -140,13 +140,8 @@ Format: `add n/NAME p/PHONE_NUMBER [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL] [a/ADDRES
...
-* `ADDRESS`, if specified, should only contain alphanumeric characters, spaces, and the following punctuations: **!"#$&'()\*+,-.:;<=>?@**. It should not be blank and be at most 80 characters long.
-* `EMAIL`, if specified, should be a **valid** email with length not exceeding 60 characters.
-* **Valid** `EMAIL` are emails that are of the format *local-part@domain* and adhere to the following constraints:
-  * The local-part should only contain alphanumeric characters and the following special characters: **+_.-**. It cannot start or end with any special characters and cannot have consecutive special characters.
-  * The domain part is made up of domain labels separated by periods (e.g. domain-label1.domain-label2.com). It must end with a domain label at least 2 characters long, each domain label starts and end with alphanumeric characters, and each domain label consists of only alphanumeric characters, separated only by hyphens, if any.
+* `EMAIL`, if specified, should be a **valid** email.
+* Similarly, `TELEGRAM` and `GITHUB`, if specified, should be **valid** usernames as well.
```
Comment (tyanhan) [inline docs/UserGuide.md:144]: I think can dont write this at all.

Comment (vishandi) [reply docs/UserGuide.md:144]: i was thinking about that, but that means name and phone number shouldn't be mentioned at all also right? As all attributes should be equivalent, so if one is mentioned, the rest should be also?

Comment (tyanhan) [reply docs/UserGuide.md:144]: Yup, I believe we can remove the name and phone ones also. Same for some of the constraints for addEvent and editEvent

**Review by tyanhan — APPROVED** [review]
LGTM!

**Pull request merged**